permissions-avfoundation/build.gradle.kts

```diff
@@ -5,32 +5,34 @@
 import dev.icerock.moko.gradle.utils.connectTargetsToSourceSet
 import dev.icerock.moko.gradle.utils.createMainTest
 import dev.icerock.moko.gradle.utils.setupDependency
+import io.gitlab.arturbosch.detekt.CONFIGURATION_DETEKT_PLUGINS
 
 plugins {
     id("org.jetbrains.kotlin.multiplatform")
     id("dev.icerock.moko.gradle.publication")
-    id("dev.icerock.moko.gradle.detekt")
+    alias(libs.plugins.detekt)
```

Comment:
in moko detekt plugin already configured all required rules and dependencies. why you not use it?